extensions/ql-vscode/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Better formatting and autoindentation when adding QLDoc comments to `.ql` and `.qll` files.
 - Allow for more flexibility when opening a database in the workspace. A user can now choose the actual database folder, or the nested `db-*` folder.
+- Add ability for users to download databases directly from LGTM.
```

Comment:
Indeed, LGTM.com only.

It would actually work for public projects on LGTM enterprise installations that have anonymous access enabled. This is a rather rare combination, so let's ignore that to avoid confusion ;-)